routers/web/repo/pull.go

```diff
@@ -217,9 +217,9 @@ func ForkPost(ctx *context.Context) {
 
 	// Check ownership of organization.
 	if ctxUser.IsOrganization() {
-		isOwner, err := models.OrgFromUser(ctxUser).IsOwnedBy(ctx.User.ID)
+		isOwner, err := models.OrgFromUser(ctxUser).CanCreateOrgRepo(ctx.User.ID)
```

Comment:
maybe change the the name of this variable. `isAllowedToFork`

Reply:
The variable name should also be changed.

Reply:
And the comment on Line 218 ;)